.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+
+  check:
+    name: Check runix
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install flox
+      uses: flox/install-flox-action@main
+      with:
+        github-access-token: ${{ secrets.NIX_GIT_TOKEN }}
+
+    - name: check clippy
+      run: flox activate -e .#runix -- cargo clippy
+
+    - name: check format
+      run: |
+        # flox activate does not activate with the current flake's self context :X
+        flox build .#rustfmt
+        export PATH="./result/bin:$PATH"
+        flox activate -e .#runix -- cargo fmt --check
```

Comment:
```suggestion
        flox develop .#runix --command cargo fmt --check
```

Reply:
Flox develop only works in interactive environments.. for some reason we abort in non interactive cases with an error message about that - considering to open a flox bug report for that.

Reply:
I believe `flox print-dev-env` should work then

Reply:
Maybe we should just tweak the error message to say to use `print-dev-env`